### PR TITLE
Add tests for withfield bytecode

### PR DESCRIPTION
**Pull request by tajila**
Add tests for withfield bytecode

Signed-off-by: tajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@llxia Can you please review this PR

File: test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java [hunk 1]
```diff
@@ -145,17 +149,64 @@ private static void initHelper(ClassWriter cw) {
 		mv.visitEnd();
 	}
 	
+	/*
+	 * This function should only be called in the 
+	 * TestWithFieldOnNonValueType test
+	 */
+	private static void testWithFieldOnNoneValueType(ClassWriter cw, String className, String[] fields) {
```
Comment (llxia) [inline test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java]: should be NonValueType?

Comment (tajila) [reply test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java]: yes, I've fixed it. 
  
I also fixed a couple other places where I made that mistake

**Comment by tajila** [comment]
https://github.com/eclipse/openj9/issues/1390

**Pull request merged**